### PR TITLE
Fix manifest.json content

### PR DESCRIPTION
**Pull request by hazae41**
Fixed app names and colors

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -10,6 +10,6 @@
   ],
   "start_url": "./index.html",
   "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "theme_color": "#0b3a53",
```
Comment (olizilla) [inline public/manifest.json]: I'm not sure where these color values get used, but the defaults set them to black and white, so I'd guess we don't want to set both of these colors to be the same.

Comment (hacdias) [reply public/manifest.json]: More info:

- https://developers.google.com/web/fundamentals/web-app-manifest/#background-color
- https://developers.google.com/web/fundamentals/web-app-manifest/#theme-color

I think it's OK to keep the same.

Comment (olizilla) [reply public/manifest.json]: Ah, thanks. Good docs!

**Pull request merged**